src/lib/updateGrantsRecipients.js

```diff
@@ -339,6 +360,14 @@ export async function processFiles(hashSumHex) {
 
       await Promise.all(grantUpdatePromises);
 
+      // Automate CDI linking to preceding recipients
+      const cdiGrantsToLink = await Grant.unscoped().findAll({
+        where: { regionId: 13, endDate: { [Op.gte]: '2021-03-17' } },
```

Comment:
I might be wrong on this suggestion but can we exclude cdi grants that already have the region id set?